TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -1411,22 +1411,23 @@ void DefaultRenderOverlay(RenderManager.CameraInfo cameraInfo)
             VehicleParked[] parkedVehiclesBuffer = Singleton<VehicleManager>.instance.m_parkedVehicles.m_buffer;
             Vehicle[] vehiclesBuffer = Singleton<VehicleManager>.instance.m_vehicles.m_buffer;
 
-            for (int i = 1; i < CitizenManager.MAX_INSTANCE_COUNT; ++i) {
-                if ((citManager.m_instances.m_buffer[i].m_flags &
-                     CitizenInstance.Flags.Character) == CitizenInstance.Flags.None) {
+            for (uint citizenInstanceId = 1; citizenInstanceId < CitizenManager.MAX_INSTANCE_COUNT; ++citizenInstanceId) {
```

Comment:
Does this get resolved at compile time? If so, will it not cause problems at later date if a mod somehow extends number of citizen instances? Should we instead query `m_length` (or whatever) on the buffer?

Reply:
Yeah, I remember there was a PR that fixed that? Or so I thought. Maybe for other constants only.

Reply:
if field is not `const` and just `readonly` then it shouldn't get resolved

Reply:
Anyway, its not part of the "ToCitizenInstance". But we could and should to a separate refactoring here to remove those constants and make a runtime call instead.